dockerize/.env

```diff
@@ -0,0 +1 @@
+JACOCODIR=./jacocoexec
```

Comment:
i think this folder would be best as called `dockerfiles`, as those are the final output files to use. can still have `scripts/dockerize`.